### PR TITLE
Remove unnecessary 'x-ua-compatible' meta tag (close #1467)

### LINKED ISSUE #1467: Remove unnecessary 'x-ua-compatible' meta tag
Starting with IE11, edge mode is the preferred document mode; it represents the highest support for modern standards available to the browser.
Use the HTML5 document type declaration to enable edge mode: <!doctype html>

Starting with IE11, document modes are deprecated and should no longer be used, except on a temporary basis. Make sure to update sites that rely on legacy features and document modes to reflect modern standards.

From: https://msdn.microsoft.com/en-us/library/bg182625(v=vs.85).aspx#docmode

### PR DESCRIPTION
**Pull request by Farfurix**

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

File: src/processing/resources/page.js [hunk 1]
```diff
@@ -118,20 +118,13 @@ class PageProcessor extends ResourceProcessorBase {
                 // TODO: Figure out how to emulate the tag behavior.
                 if (domAdapter.getAttr(meta, 'name') === 'referrer')
                     parse5Utils.setAttr(meta, 'content', 'unsafe-url');
-                // NOTE: Remove the existing ‘compatible’ meta tag and add a new one at the beginning of the head.
+                // NOTE: Remove unnecessary existing ‘compatible’ meta tag.
                 if (domAdapter.getAttr(meta, 'http-equiv') === 'X-UA-Compatible')
```
Comment (miherlosev) [inline src/processing/resources/page.js]: I propose to not remove an existing `meta` tag.
In this case we will not change nodes order.

**Comment by Farfurix** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@LavrovArtem review too
see https://github.com/DevExpress/testcafe-hammerhead/issues/1467#issue-293156297

**Pull request merged**